server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -258,7 +271,7 @@ export default class BulkChangeModal extends React.PureComponent {
       {affected != null && this.renderAffected(affected)}
     </div>;
 
-  renderAssigneeOption = (option: { avatar?: string, email?: string, label: string }) =>
+  renderAssigneeOption = (option /*: { avatar?: string, email?: string, label: string } */) =>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)